src/sentry/tasks/deletion.py

```diff
@@ -307,12 +341,15 @@ def delete_objects(models, relation, transaction_id=None, limit=100, logger=None
     # This handles cascades properly
     has_more = False
     for model in models:
-        if logger is not None:
-            logger.info('remove.%s' % model.__name__.lower(), extra=dict(
-                relation.items() + [('transaction_id', transaction_id)],
-            ))
         for obj in model.objects.filter(**relation)[:limit]:
+            obj_id = obj.id
+            slug = type(obj).__name__.lower()
```

Comment:
This took me a moment, since this isn't a `slug`, it's the model name. I'd just suggest renaming here since I was thinking it was an instance slug or something.